src/main/java/no/api/freemarker/java8/EnhancedJava8ObjectWrapper.java

```diff
@@ -0,0 +1,32 @@
+package no.api.freemarker.java8;
+
+import freemarker.template.TemplateHashModel;
+import freemarker.template.TemplateModel;
+import freemarker.template.TemplateModelException;
+import freemarker.template.Version;
+import no.api.freemarker.java8.time.TemporalDialerAdapter;
+import no.api.freemarker.java8.zone.ZonedDateTimeStrategy;
+
+import java.time.temporal.Temporal;
+
+/**
+ * Freemarker ObjectWrapper that extends the Java8ObjectWrapper with support for time manipulations in all temporal classes in the new java.time api.
+ * Such time manipulation operations include plusSeconds, plusMinutes, etc.
+ */
+public class EnhancedJava8ObjectWrapper extends Java8ObjectWrapper {
```

Comment:
Why this Enhanced wrapper ? Couldn't we just add it to the Java8ObjetWrapper ?

Reply:
Of course. I didn't want to be too intrusive. I actually more prefer to merge the changes into Java8ObjetWrapper. Will modify PR soon

Reply:
Done. Please review

Reply:
Thank! I will look into this and merge it soon. I think v2 should be released within a week or so.